c10/cuda/CUDACachingAllocator.h

```diff
@@ -39,6 +40,67 @@ namespace cuda {
 
 namespace CUDACachingAllocator {
 
+struct Stat {
+  int64_t current = 0;
+  int64_t peak = 0;
+  int64_t allocated = 0;
```

Comment:
It's kind of weird to call this field "allocated" when you're using it to describe things like "inactive split". but tbh I don't know a better name.

Reply:
I was thinking of `total_created` and `total_destroyed`, but could be even more confusing :P